### ISSUE TITLE
Improve Verification HTML Pages

### ISSUE THREAD

**Issue by cycomachead** [bromagosa/snapCloud#118]
Decided that this should be done separately from #117 

The existing `htmlPage` structure makes a bit difficult to be more verbose. 
Is there a problem with using etlua views?

**Comment by bromagosa** [comment]
> Is there a problem with using etlua views?

Yes, these simple html pages are meant to be temporary until we publish the social site. I don't want to add any views at all to the backend at all. My idea from the beginning was that the backend would be just a plain and simple server-side API :)

If you think it's wise to do so, we could maybe redirect these pages to new pages in the social site right now. I'm scared people will start using it before we've had the time to fix a bunch of serious issues though.

Otherwise, temporary ugly string-concatenated HTML pages do the trick and are super easy to strip out when we do the switch.

**Comment by cycomachead** [comment]
Yeah, I guess I was assuming we'd just temporarily use etlua views, but not stick with them full time?

I had the same reservations about linking to the social site too much.
